test/interface.spec.js

```diff
@@ -0,0 +1,235 @@
+/* eslint-env mocha */
+'use strict'
+
+const tests = require('interface-ipfs-core')
+const isNode = require('detect-node')
+const CommonFactory = require('./utils/interface-common-factory')
+const IPFSApi = require('../src')
+const isWindows = process.platform && process.platform === 'win32'
+
+describe('interface-ipfs-core tests', () => {
+  const defaultCommonFactory = CommonFactory.create()
+
+  tests.bitswap(defaultCommonFactory, {
+    skip: [
+      // bitswap.unwant
+      {
+        name: 'should remove a key from the wantlist',
+        reason: 'FIXME why is this skipped?'
```

Comment:
@alanshaw does it fail when run?

Reply:
Yes unfortunately. It was just skipped in interface-ipfs-core before with no explanation.

```
  1) interface-ipfs-core tests
       .bitswap.unwant
         should remove a key from the wantlist:

      Uncaught AssertionError: expected false to equal true
      + expected - actual

      -false
      +true
      
      at ipfsB.bitswap.wantlist (/Users/alan/Code/protocol-labs/interface-ipfs-core/js/src/bitswap/unwant.js:52:61)
      at f (node_modules/once/once.js:25:25)
      at streamToValue (src/utils/stream-to-json-value.js:30:5)
      at concat (src/utils/stream-to-value.js:12:22)
      at ConcatStream.<anonymous> (node_modules/concat-stream/index.js:37:43)
      at finishMaybe (node_modules/readable-stream/lib/_stream_writable.js:630:14)
      at afterWrite (node_modules/readable-stream/lib/_stream_writable.js:492:3)
      at process._tickCallback (internal/process/next_tick.js:63:19)
```